libsimba/utils.py

```diff
@@ -248,3 +246,29 @@ def build_url(base_api_url: str, path: str, query_dict: Optional[dict]):
     url_parts[2] = path
     url_parts[4] = urlencode(query_dict)
     return urlunparse(url_parts)
+
+def get_address(deployment: dict) -> Optional[str]:
+    """
+    Extract the primary address from a deployment object.
+
+    :param deployment: The deployment object
+    :type deployment: dict
+    :return: Optional[str]
+    """
+    primary = deployment.get("primary")
+    if primary:
+        return primary.get("address")
+    return None
+
+def get_deployed_artifact_id(deployment: dict) -> Optional[str]:
+    """
+    Extract the primary deployed artifact ID from a deployment object.
+
+    :param deployment: The deployment object
+    :type deployment: dict
+    :return: Optional[str]
+    """
+    primary = deployment.get("primary")
```

Comment:
This can be
return deployment.get("primary", {}).get("deployed_artifact_id", None)
Same for the other one... in fact maybe this can be remove?

Reply:
done